### ISSUE TITLE
mysqld_exporter: error: unexpected SyslogIdentifier=mysqld_exporter

### ISSUE THREAD

**Issue by lunarthegrey** [cloudalchemy/ansible-mysqld_exporter#21]
It seems on Debian 10, this error causes the service to fail when starting. It works fine on CentOS 7 with the same systemd service which is strange. Any ideas?

```
Sep 27 00:10:19 db-prod-01 mysqld_exporter[11581]: mysqld_exporter: error: unexpected SyslogIdentifier=mysqld_exporter, try --help
Sep 27 00:10:19 db-prod-01 systemd[1]: mysqld_exporter.service: Main process exited, code=exited, status=1/FAILURE
Sep 27 00:10:19 db-prod-01 systemd[1]: mysqld_exporter.service: Failed with result 'exit-code'.
```

**Comment by lunarthegrey** [comment]
It seems like it complains about most of the options in the service. Really strange... For example, I comment out `SyslogIdentifier`

```
Sep 27 00:28:49 db-prod-01 mysqld_exporter[14422]: mysqld_exporter: error: unexpected Restart=always, try --help
Sep 27 00:28:49 db-prod-01 systemd[1]: mysqld_exporter.service: Main process exited, code=exited, status=1/FAILURE
Sep 27 00:28:49 db-prod-01 systemd[1]: mysqld_exporter.service: Failed with result 'exit-code'.
```

And it's on to the next line in the service file..

**Comment by paulfantom** [comment]
Could you paste full content of `/etc/systemd/system/mysqld_exporter.service` from target host?

**Comment by lunarthegrey** [comment]
@paulfantom, thanks for responding. Here it is.

```
[Unit]
Description=Prometheus Mysqld Exporter
After=network-online.target
StartLimitInterval=0

[Service]
Type=simple
User=mysqld-exp
Group=mysqld-exp
Environment="DATA_SOURCE_NAME=exporter:gUHG1ugHuSzs@(localhost:3306)/"
ExecStart=/usr/local/bin/mysqld_exporter \
    --web.listen-address 0.0.0.0:9104 \

SyslogIdentifier=mysqld_exporter
Restart=always
RestartSec=1

[Install]
WantedBy=multi-user.target
```

**Comment by lunarthegrey** [comment]
Exact same file as the one I'm seeing on my CentOS 7 server.

**Comment by paulfantom** [comment]
You have stray `\` at the end of `--web.listen-address 0.0.0.0:9104 \` line. This is causing invalid systemd service definition. Issue is already fixed on master branch with this commit: https://github.com/cloudalchemy/ansible-mysqld_exporter/commit/059558cdbb9226806abcb66aef263d633f35ef11 and will be soon available in version 0.3.0 of this exporter.

**Comment by lunarthegrey** [comment]
Thanks @paulfantom. Strange as I have the `\` on my CentOS server as well. I guess different versions of systemd tolerate it differently.
